src/pages/DeckIndex.js

```diff
@@ -1,9 +1,11 @@
 import React from "react";
-import { NavLink } from "react-router-dom"
+import { NavLink, useParams } from "react-router-dom"
+import './Deckindex.css' 
 import { Button } from "reactstrap";
 import '../App.css' 
 
 const DeckIndex = ({ decks }) => {
+  const {deck_id} = useParams()
```

Comment:
I don't see this being utilized in this component.  Are there any params for DeckIndex?